src/encoded/tests/data/inserts/file_reference.json

```diff
@@ -126,24 +126,45 @@
      "filename": "test_fasta2.fasta.gz"
  },
  {
-   "uuid": "9c78392f-d183-418f-8cbb-eafe544b9de0",
-   "description": "This file is used to see the annotations in higlass",
-   "filename": "gene-annotations-hg38.beddb",
+    "uuid": "9c78392f-d183-418f-8cbb-eafe544b9de0",
+    "description": "This file is used to see the annotations in higlass",
+    "filename": "gene-annotations-hg38.beddb",
+    "file_classification": "ancillary file",
+    "higlass_uid": "XGrAqZupQb2h5Pjiarf8PA",
+    "award": "1U01CA200059-01",
+    "status": "released",
+    "file_format": "76dc8c06-67d8-487b-8fc8-d841752a0b60",
+    "tags": [
+        "higlass_reference"
+    ],
+    "accession": "4DNFIWG6CQQA",
+    "file_size": 11776000,
+    "lab": "4dn-dcic-lab",
+    "submitted_by": "4dndcic@gmail.com",
+    "genome_assembly": "GRCh38",
+    "aliases": [
+        "4dn-dcic-lab:higlass_annotation_hg38"
+    ]
+ },
+ {
+   "uuid": "9866d158-da3c-4d9b-96a9-1d59632eabeb",
+   "description": "Full chromosome sizes file for the human genome reference GRCh38",
    "file_classification": "ancillary file",
-   "higlass_uid": "XGrAqZupQb2h5Pjiarf8PA",
+   "aliases": [
+     "4dn-dcic-lab:GRCh38_chromsizes_full"
+   ],
+   "accession": "4DNFIZJB62D1",
+   "filename": "GRCh38.full.chrom.sizes",
+   "higlass_uid": "NThJBk8mSnqaTzeDaWL-kg",
```

Comment:
"higlass_uid": "JXbq7f-GTeq3FJy_ilIomQ" will use the chromsize file without the unplaced contigs (ugly things at the end of chromsizes grid)

Reply:
Thanks Carl.
You can't see the grid here because HiGlass drew it below the heatmap (the 2d-chromosome-grid is the first track listed in the viewconfig you revealed.) I missed this setup, so I'll fix that now.

HiGlass uid has been updated.